plugins/jobs/src/js/components/JobsOverviewTable.js

```diff
@@ -22,6 +22,10 @@ import JobTableHeaderLabels from "../constants/JobTableHeaderLabels";
 
 const METHODS_TO_BIND = ["renderHeadline", "jobSortFunction"];
 
+const JobsCronTooltip = lazy(() =>
```

Comment:
Lazy-loading this component because it depends on `prettycron`, which depends on `moment`.

I tried using [cronstrue](https://github.com/bradymholt/cRonstrue), but it uses different language than `moment`.

moment: https://cl.ly/69a125629a43
cronstrue: https://cl.ly/3e019537d16f

Reply:
i don't know how crucial it is to change language, but i actually like the cronstrue-format better. also it's only 4kb when gzipped and does not bring a lot of unneeded features (prettycron also brings `later`, which in turn even forces [some webpack-specifics](https://github.com/dcos/dcos-ui/blob/411b5f11f29c5d81a67e1b09dcb6c18a8086f148/webpack/webpack.config.js#L81) on us) . :)

Reply:
Yea, I like the cronstrue format better too. I just wasn't sure if we were ok with changing that in this ticket. I'd love to hear from others :)

Reply:
Who do we have to convince? product or design? I like cronstrue's format better, plus it has internationalization options.

Reply:
@pierrebeitz and @brandonc:
I talked to the designers and Tobi in our Product Design sync today. They're ok with the cronstrue style of formatting.

I filed a ticket to follow-up on this: https://jira.mesosphere.com/browse/DCOS-51104